src/client.ts

```diff
@@ -1320,6 +1320,40 @@ export class MatrixClient extends EventEmitter {
         return this.syncApi.getSyncStateData();
     }
 
+    /**
+     * Main entry point for starting sync.
+     */
+    public startSync(): void {
+        if (this.syncApi == null) {
+            logger.error("Calling sync() while no sync object, this should not happen, cannot start.");
+            return;
+        }
+        this.syncApi.sync();
+    }
```

Comment:
SDK-wise, I'm not sure we want to support a "start sync" function - the consumer should be starting the whole client instance, not just sync.